src/components/modules/forms/PublishAssets/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { Formik, Form, Field } from "formik";
+import { initialPublishAssetsValues } from "./publishAssets.constants";
+import { useAssets } from "./useAssets";
+
+export const PublishAssetsForm = () => {
+    const { publishAsset, handleFileLoad } = useAssets();
+
+    return (
+        <Formik initialValues={initialPublishAssetsValues} onSubmit={publishAsset}>
+            {({ setFieldValue }) => (
+                <Form>
+                    <Field type={'text'} name={'title'} placeholder={'Title'} />
+                    <input type={'file'} name={'file'} onChange={e => handleFileLoad(e, setFieldValue)} />
```

Comment:
All the form inputs should be wrapper of "Field" imported from formik, then you can use there value, error handling etc.